demos/linear_regression/linreg_setup.py

```diff
@@ -0,0 +1,342 @@
+# Copyright 2021 Fedlearn authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example code of federated linear regression client based on QR decompositon.
+
+
+Example
+-------
+TBA::
+
+    $ TBA
+
+
+"""
+
+import setup
+setup.deal_with_path()
+
+import numpy as np
+import qrClient as qrClient
+import qrCoordinator as qrCoord
+import socket
+import pandas
+from core.entity.common.machineinfo import MachineInfo
+
+
+def generate_fullrank_test_data(nTrain, nInfer, nFeatures, dataMax=10**4):
```

Comment:
If these functions are used for data generation, I suggest just keep them in another place. For real problems there are no needs to fake data.